module/user-extension/helper.js

```diff
@@ -681,8 +681,51 @@ module.exports = class UserExtensionHelper {
             }
         })
     }
+    
+   /**
+   * bulkWrite the userExtension collection
+   * @method
+   * @name bulkWrite
+   * @param {Array} query - Array of queries to update.
+   * @returns {Array} 
+   */
+   
+    static bulkWrite(query){
+        return new Promise(async (resolve, reject) => {
+            try{
+                let insertProgram = await database.models.userExtension.bulkWrite(query)
+                return resolve(insertProgram)
+            }catch(error){
+                reject(error);
+            }
+        })
+    }
+
+   /**
+   * update the userExtension document.
+   * @method
+   * @name updateOne
+   * @param {Object} query    - findMatchQuery.
```

Comment:
@praveenKDass for all functions make sure function signature i.e. argument and return values are well described and understandable.